<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsWindow</class>
 <widget class="QWidget" name="ResultsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>面向SAR图像舰船目标的多尺度轻量化检测研究-检测结果  Shi Kequan</string>
  </property>
  <property name="windowIcon">
   <iconset resource="YOLOSHOWUI.qrc">
    <normaloff>:/jiaoda/images/physics2.png</normaloff>:/jiaoda/images/physics2.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="table_widget">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>图片名字</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>图片尺寸</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>检测内容</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>推理速度 (ms)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="remove_button">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_button">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="YOLOSHOWUI.qrc"/>
 </resources>
 <connections/>
</ui>
